packages/sanity/src/core/form/inputs/PortableText/object/Annotation.tsx

```diff
@@ -281,28 +282,50 @@ export const DefaultAnnotationComponent = (props: BlockAnnotationProps) => {
     return 'default'
   }, [isLink, hasError, hasWarning])
 
+  const handleOnClick = useCallback(() => {
+    setShowPopover(true)
+  }, [])
+
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if (event.key === 'Escape') {
+        setShowPopover(true)
+      }
+    }
+
+    document.addEventListener('keydown', handleKeyDown)
+
+    return () => {
+      document.removeEventListener('keydown', handleKeyDown)
+    }
+  }, [])
+
```

Comment:
This should be dropped. It's taken care [of here](https://github.com/sanity-io/sanity/blob/edx-813/packages/sanity/src/core/form/inputs/PortableText/object/AnnotationToolbarPopover.tsx#L80)

Reply:
I can't see any different behaviour when removing it, but maybe I'm missing something?

Reply:
No, you're right 🤔